company_code/readme/USAGE.rst

```diff
@@ -0,0 +1,11 @@
+To display your company code in with `name_get()` just 
+produce this code in your custom according your model
+
```

Comment:
s/in with/with/
s/just produce/just write/
s/in your custom/in your custom code/